src/main/resources/templates/build.gradle.ftl

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This is a general purpose Gradle build.
+ * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds/
+ */
+<#if  language == "kotlin">
+buildscript {
```

Comment:
You can avoid a `buildScript` block:

```groovy
plugins {
    id "org.jetbrains.kotlin.jvm" version "1.3.11"
}
```

see https://kotlinlang.org/docs/reference/using-gradle.html